src/style.css

```diff
@@ -1,7 +1,49 @@
-.hello {
-    color: red;
-  }
 
-  body {
-    background-color: bisque;
-}
\ No newline at end of file
+body {
+  width: 50%;
+  margin: auto;
+  border: 1px solid #ccc;
+  height: 50vh;
+  padding: 20px;
+  margin-top: 50px;
+}
+
+.main {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 30px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.refresh {
+  background-color: #f1f1f1;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  padding: 10px;
+  cursor: pointer;
+  height: 40px;
+}
+
+.error,
+.success {
+  margin-top: 20px;
+  width: 35%;
+  color: white;
+  display: flex;
+  justify-content: center;
+  border-radius: 10px;
+  padding: 10px;
+  box-shadow: 1px 1px 1px 1px black;
```

Comment:
- [ ] According to the project requirements, the App must be properly styled and its layout should look like the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png). Kindly improve this 🎨 🖌️ and remember, The Style is your choice and it does not need to be perfect. They just need to keep the general layout from the wireframe.

![image](https://user-images.githubusercontent.com/79058364/219925090-4ed0e89a-3a62-451c-a35d-44c6f7e066af.png)

Reply:
Hello, @BenMukebo I will make the changes. Since I only have 25 minutes. could you stay while I make the changes, so you can review them very fast, please?